src/models/relationshipAttributeSet.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2021  Akash Gupta
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import {camelToSnake, snakeToCamel} from '../util';
+
+
+export default function relationshipAttributeSet(bookshelf) {
+	const RelationshipAttributeSet = bookshelf.Model.extend({
+		attribute() {
```

Comment:
I don't know why but naming it as `attributes` gives **Error: attributes is not defined on the model**. So I had to use **attribute** instead.

Reply:
Yeah, I investigated a bit, that's unfortunate but I think we can't use attributes.
I still think we should try for something else plural, it's confusing otherwise.
Maybe `relationshipAttributes` ?

Reply:
sounds good to me :+1: